  INFO root [2018-06-21 09:33:04,482] - 
  INFO root [2018-06-21 09:33:04,482] - -----------------------------------------------------------------------------------------


  INFO root [2018-06-21 09:33:04,482] - Resmgr::initialize: KBE_ROOT=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/
  INFO root [2018-06-21 09:33:04,492] - Resmgr::initialize: KBE_RES_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/res/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/res/
  INFO root [2018-06-21 09:33:04,492] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/bin/server/
  INFO root [2018-06-21 09:33:04,512] - NetworkInterface::recreateListeningSocket(EXTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:20013 )
  INFO root [2018-06-21 09:33:04,512] - NetworkInterface::recreateListeningSocket(EXTERNAL): address 192.168.31.56:20013, SOMAXCONN=128.
  INFO root [2018-06-21 09:33:04,512] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:2616 )
  INFO root [2018-06-21 09:33:04,522] - NetworkInterface::recreateListeningSocket(INTERNAL): address 192.168.31.56:2616, SOMAXCONN=128.
  INFO root [2018-06-21 09:33:04,522] - server-configs:
  INFO root [2018-06-21 09:33:04,522] - 	internalAddr : 192.168.31.56:2616
  INFO root [2018-06-21 09:33:04,522] - 	externalAddr : 192.168.31.56:20013
  INFO root [2018-06-21 09:33:04,523] - 	componentID : 9000
  INFO root [2018-06-21 09:33:06,374] - ---- loginapp Version: 1.0.0. ScriptVersion: 0.1.0. Protocol: A16A7477E55D6AEDCD48345D2B50A665. Config: Debug X86. Built: 23:20:06 Jun 15 2018. AppID: 9000. UID: 23460. PID: 10408 ----
  INFO root [2018-06-21 09:33:06,374] - Server :        Intel(R) Core(TM) i5-2415M CPU @ 2.30GHz with 4194303kB RAM
  INFO root [2018-06-21 09:33:06,374] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2018-06-21 09:33:06,374] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [2018-06-21 09:33:06,544] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [2018-06-21 09:33:06,544] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [2018-06-21 09:33:06,544] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [2018-06-21 09:33:06,554] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [2018-06-21 09:33:06,555] - Script::install_py_dlls(): PyInit_select
 DEBUG root [2018-06-21 09:33:06,555] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [2018-06-21 09:33:06,557] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [2018-06-21 09:33:06,671] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [2018-06-21 09:33:06,822] - Script::install(): is successfully, Python=(3.4.2 (default, Jun 15 2018, 23:20:02) [MSC v.1913 32 bit (Intel)])!
  INFO root [2018-06-21 09:33:06,855] - Script::install(): KBExtra is successfully!
S_INFO root [2018-06-21 09:33:06,860] - onLoginAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2018-06-21 09:33:06,861] - TelnetServer server is running on port 31000
  INFO root [2018-06-21 09:33:07,984] - ---- loginapp is running ----
 DEBUG root [2018-06-21 09:33:07,984] - Components::process(): Request for the process of identity...
  INFO root [2018-06-21 09:33:15,086] - Components::findComponents: find logger(1)...
  INFO root [2018-06-21 09:33:15,176] - Components::findComponents: found logger, addr:192.168.31.56:2624
  INFO root [2018-06-21 09:33:15,176] - Components::addComponent: reset orderLog, uid=23460!
  INFO root [2018-06-21 09:33:15,176] - Components::addComponent[logger], uid=23460, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2018-06-21 09:33:15,176] - Components::findComponents: find dbmgr(2)...
  WARN root [2018-06-21 09:33:16,276] - Forwarding logs to logger[192.168.31.56:2624/0/0/0]...
 ERROR root [2018-06-21 09:33:33,820] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [2018-06-21 09:33:33,823] - The following logs sent to logger failed:
  WARN root [2018-06-21 09:33:33,823] - ==> [2018-06-21 09:33:33 819] TCPPacketReceiver::processPendingEvents(192.168.31.56:2624): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2018-06-21 09:33:33,824] - ==> [2018-06-21 09:33:33 821] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [2018-06-21 09:33:34,080] - Components::findComponents: find dbmgr(11)...
  INFO root [2018-06-21 09:34:04,523] - 
  INFO root [2018-06-21 09:34:04,525] - -----------------------------------------------------------------------------------------


  INFO root [2018-06-21 09:34:04,527] - Resmgr::initialize: KBE_ROOT=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/
  INFO root [2018-06-21 09:34:04,528] - Resmgr::initialize: KBE_RES_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/res/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/res/
  INFO root [2018-06-21 09:34:04,528] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/bin/server/
  INFO root [2018-06-21 09:34:04,541] - NetworkInterface::recreateListeningSocket(EXTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:20013 )
  INFO root [2018-06-21 09:34:04,542] - NetworkInterface::recreateListeningSocket(EXTERNAL): address 192.168.31.56:20013, SOMAXCONN=128.
  INFO root [2018-06-21 09:34:04,546] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:2659 )
  INFO root [2018-06-21 09:34:04,546] - NetworkInterface::recreateListeningSocket(INTERNAL): address 192.168.31.56:2659, SOMAXCONN=128.
  INFO root [2018-06-21 09:34:04,546] - server-configs:
  INFO root [2018-06-21 09:34:04,547] - 	internalAddr : 192.168.31.56:2659
  INFO root [2018-06-21 09:34:04,547] - 	externalAddr : 192.168.31.56:20013
  INFO root [2018-06-21 09:34:04,547] - 	componentID : 9000
  INFO root [2018-06-21 09:34:05,492] - Components::findLogger: found logger, addr:192.168.31.56:2656
  INFO root [2018-06-21 09:34:05,492] - Components::addComponent: reset orderLog, uid=23656!
  INFO root [2018-06-21 09:34:05,492] - Components::addComponent[logger], uid=23656, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2018-06-21 09:34:05,512] - ---- loginapp Version: 1.0.0. ScriptVersion: 0.1.0. Protocol: A16A7477E55D6AEDCD48345D2B50A665. Config: Debug X86. Built: 23:20:06 Jun 15 2018. AppID: 9000. UID: 23656. PID: 4416 ----
  INFO root [2018-06-21 09:34:05,512] - Server :        Intel(R) Core(TM) i5-2415M CPU @ 2.30GHz with 4194303kB RAM
  INFO root [2018-06-21 09:34:05,512] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2018-06-21 09:34:05,512] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [2018-06-21 09:34:05,652] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [2018-06-21 09:34:05,652] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [2018-06-21 09:34:05,652] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [2018-06-21 09:34:05,652] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [2018-06-21 09:34:05,652] - Script::install_py_dlls(): PyInit_select
 DEBUG root [2018-06-21 09:34:05,652] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [2018-06-21 09:34:05,662] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [2018-06-21 09:34:05,732] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [2018-06-21 09:34:05,802] - Script::install(): is successfully, Python=(3.4.2 (default, Jun 15 2018, 23:20:02) [MSC v.1913 32 bit (Intel)])!
  INFO root [2018-06-21 09:34:05,802] - Script::install(): KBExtra is successfully!
S_INFO root [2018-06-21 09:34:05,802] - onLoginAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2018-06-21 09:34:05,802] - TelnetServer server is running on port 31000
  INFO root [2018-06-21 09:34:07,254] - ---- loginapp is running ----
 DEBUG root [2018-06-21 09:34:07,256] - Components::process(): Request for the process of identity...
  WARN root [2018-06-21 09:34:13,386] - Forwarding logs to logger[192.168.31.56:2656/0/0/0]...
 ERROR root [2018-06-21 09:40:44,053] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [2018-06-21 09:40:44,058] - The following logs sent to logger failed:
  WARN root [2018-06-21 09:40:44,059] - ==> [2018-06-21 09:40:44 049] TCPPacketReceiver::processPendingEvents(192.168.31.56:2656): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2018-06-21 09:40:44,059] - ==> [2018-06-21 09:40:44 054] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [2018-06-21 09:40:44,152] - TCPPacketReceiver::processPendingEvents(192.168.31.56:2660): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2018-06-21 09:40:44,159] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [2018-06-21 09:40:44,168] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [2018-06-21 09:40:44,169] - onLoginAppShutDown()
  INFO root [2018-06-21 09:40:44,169] - Shutdowner::onShutdown: shutting down
  INFO root [2018-06-21 09:40:44,169] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2018-06-21 09:40:44,170] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [2018-06-21 09:40:44,170] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [2018-06-21 09:40:44,170] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [2018-06-21 09:40:44,171] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [2018-06-21 09:40:44,171] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [2018-06-21 09:40:44,171] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [2018-06-21 09:40:44,172] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [2018-06-21 09:40:44,172] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [2018-06-21 09:40:44,190] - Script::uninstall(): is successfully!
 DEBUG root [2018-06-21 09:40:44,191] - ThreadPool::destroy(): starting size 2.
  WARN root [2018-06-21 09:40:44,491] - ThreadPool::destroy(): waiting for thread(2)[0x5f0190,0x5f0128,], try=1
  INFO root [2018-06-21 09:40:52,093] - 
  INFO root [2018-06-21 09:40:52,094] - -----------------------------------------------------------------------------------------


  INFO root [2018-06-21 09:40:52,096] - Resmgr::initialize: KBE_ROOT=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/
  INFO root [2018-06-21 09:40:52,096] - Resmgr::initialize: KBE_RES_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/res/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/res/
  INFO root [2018-06-21 09:40:52,097] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/bin/server/
  INFO root [2018-06-21 09:40:52,116] - NetworkInterface::recreateListeningSocket(EXTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:20013 )
  INFO root [2018-06-21 09:40:52,117] - NetworkInterface::recreateListeningSocket(EXTERNAL): address 192.168.31.56:20013, SOMAXCONN=128.
  INFO root [2018-06-21 09:40:52,123] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:2758 )
  INFO root [2018-06-21 09:40:52,124] - NetworkInterface::recreateListeningSocket(INTERNAL): address 192.168.31.56:2758, SOMAXCONN=128.
  INFO root [2018-06-21 09:40:52,124] - server-configs:
  INFO root [2018-06-21 09:40:52,124] - 	internalAddr : 192.168.31.56:2758
  INFO root [2018-06-21 09:40:52,125] - 	externalAddr : 192.168.31.56:20013
  INFO root [2018-06-21 09:40:52,125] - 	componentID : 9000
  INFO root [2018-06-21 09:40:54,101] - ---- loginapp Version: 1.0.0. ScriptVersion: 0.1.0. Protocol: A16A7477E55D6AEDCD48345D2B50A665. Config: Debug X86. Built: 23:20:06 Jun 15 2018. AppID: 9000. UID: 24988. PID: 8356 ----
  INFO root [2018-06-21 09:40:54,103] - Server :        Intel(R) Core(TM) i5-2415M CPU @ 2.30GHz with 4194303kB RAM
  INFO root [2018-06-21 09:40:54,103] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2018-06-21 09:40:54,104] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [2018-06-21 09:40:54,339] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [2018-06-21 09:40:54,341] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [2018-06-21 09:40:54,343] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [2018-06-21 09:40:54,350] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [2018-06-21 09:40:54,350] - Script::install_py_dlls(): PyInit_select
 DEBUG root [2018-06-21 09:40:54,351] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [2018-06-21 09:40:54,352] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [2018-06-21 09:40:54,536] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [2018-06-21 09:40:54,738] - Script::install(): is successfully, Python=(3.4.2 (default, Jun 15 2018, 23:20:02) [MSC v.1913 32 bit (Intel)])!
  INFO root [2018-06-21 09:40:54,738] - Script::install(): KBExtra is successfully!
S_INFO root [2018-06-21 09:40:54,743] - onLoginAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2018-06-21 09:40:54,744] - TelnetServer server is running on port 31000
  INFO root [2018-06-21 09:40:55,654] - ---- loginapp is running ----
 DEBUG root [2018-06-21 09:40:55,654] - Components::process(): Request for the process of identity...
  INFO root [2018-06-21 09:41:02,831] - Components::findComponents: find logger(1)...
  INFO root [2018-06-21 09:41:02,831] - Components::findComponents: found logger, addr:192.168.31.56:2756
  INFO root [2018-06-21 09:41:02,831] - Components::addComponent: reset orderLog, uid=24988!
  INFO root [2018-06-21 09:41:02,831] - Components::addComponent[logger], uid=24988, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2018-06-21 09:41:02,831] - Components::findComponents: find dbmgr(2)...
  INFO root [2018-06-21 09:41:02,831] - Components::findComponents: found dbmgr, addr:192.168.31.56:2754
  INFO root [2018-06-21 09:41:02,831] - Components::addComponent: reset orderLog, uid=24988!
  INFO root [2018-06-21 09:41:02,831] - Components::addComponent[dbmgr], uid=24988, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2018-06-21 09:41:02,831] - Components::findComponents: found dbmgr, addr:192.168.31.56:2754
  WARN root [2018-06-21 09:41:03,951] - Forwarding logs to logger[192.168.31.56:2756/0/0/0]...
 ERROR root [2018-06-21 22:33:34,599] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [2018-06-21 22:33:34,599] - The following logs sent to logger failed:
  WARN root [2018-06-21 22:33:34,599] - ==> [2018-06-21 22:33:33 805] TCPPacketReceiver::processPendingEvents(192.168.31.56:2756): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2018-06-21 22:33:34,599] - ==> [2018-06-21 22:33:34 753] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [2018-06-21 22:33:34,599] - TCPPacketReceiver::processPendingEvents(192.168.31.56:2754): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2018-06-21 22:33:34,599] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [2018-06-21 22:33:34,599] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2018-06-21 22:33:43,152] - 
  INFO root [2018-06-21 22:33:43,153] - -----------------------------------------------------------------------------------------


  INFO root [2018-06-21 22:33:43,155] - Resmgr::initialize: KBE_ROOT=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/
  INFO root [2018-06-21 22:33:43,156] - Resmgr::initialize: KBE_RES_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/res/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/res/
  INFO root [2018-06-21 22:33:43,156] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/bin/server/
  INFO root [2018-06-21 22:33:43,178] - NetworkInterface::recreateListeningSocket(EXTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:20013 )
  INFO root [2018-06-21 22:33:43,179] - NetworkInterface::recreateListeningSocket(EXTERNAL): address 192.168.31.56:20013, SOMAXCONN=128.
  INFO root [2018-06-21 22:33:43,183] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:10048 )
  INFO root [2018-06-21 22:33:43,183] - NetworkInterface::recreateListeningSocket(INTERNAL): address 192.168.31.56:10048, SOMAXCONN=128.
  INFO root [2018-06-21 22:33:43,184] - server-configs:
  INFO root [2018-06-21 22:33:43,184] - 	internalAddr : 192.168.31.56:10048
  INFO root [2018-06-21 22:33:43,184] - 	externalAddr : 192.168.31.56:20013
  INFO root [2018-06-21 22:33:43,184] - 	componentID : 9000
  INFO root [2018-06-21 22:33:45,214] - ---- loginapp Version: 1.0.0. ScriptVersion: 0.1.0. Protocol: A16A7477E55D6AEDCD48345D2B50A665. Config: Debug X86. Built: 23:20:06 Jun 15 2018. AppID: 9000. UID: 12576. PID: 14268 ----
  INFO root [2018-06-21 22:33:45,215] - Server :        Intel(R) Core(TM) i5-2415M CPU @ 2.30GHz with 4194303kB RAM
  INFO root [2018-06-21 22:33:45,215] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2018-06-21 22:33:45,216] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [2018-06-21 22:33:45,410] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [2018-06-21 22:33:45,411] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [2018-06-21 22:33:45,412] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [2018-06-21 22:33:45,420] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [2018-06-21 22:33:45,421] - Script::install_py_dlls(): PyInit_select
 DEBUG root [2018-06-21 22:33:45,421] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [2018-06-21 22:33:45,423] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [2018-06-21 22:33:45,552] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [2018-06-21 22:33:45,682] - Script::install(): is successfully, Python=(3.4.2 (default, Jun 15 2018, 23:20:02) [MSC v.1913 32 bit (Intel)])!
  INFO root [2018-06-21 22:33:45,682] - Script::install(): KBExtra is successfully!
S_INFO root [2018-06-21 22:33:45,688] - onLoginAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2018-06-21 22:33:45,690] - TelnetServer server is running on port 31000
  INFO root [2018-06-21 22:33:46,790] - ---- loginapp is running ----
 DEBUG root [2018-06-21 22:33:46,791] - Components::process(): Request for the process of identity...
  INFO root [2018-06-21 22:33:53,909] - Components::findComponents: find logger(1)...
  INFO root [2018-06-21 22:33:54,008] - Components::findComponents: found logger, addr:192.168.31.56:10047
  INFO root [2018-06-21 22:33:54,009] - Components::addComponent: reset orderLog, uid=12576!
  INFO root [2018-06-21 22:33:54,009] - Components::addComponent[logger], uid=12576, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2018-06-21 22:33:54,011] - Components::findComponents: find dbmgr(2)...
  INFO root [2018-06-21 22:33:54,111] - Components::findComponents: found dbmgr, addr:192.168.31.56:10045
  INFO root [2018-06-21 22:33:54,111] - Components::addComponent: reset orderLog, uid=12576!
  INFO root [2018-06-21 22:33:54,112] - Components::addComponent[dbmgr], uid=12576, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2018-06-21 22:33:54,112] - Components::findComponents: found dbmgr, addr:192.168.31.56:10045
  WARN root [2018-06-21 22:33:55,112] - Forwarding logs to logger[192.168.31.56:10047/0/0/0]...
 ERROR root [2018-06-21 22:33:55,526] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [2018-06-21 22:33:55,527] - The following logs sent to logger failed:
  WARN root [2018-06-21 22:33:55,527] - ==> [2018-06-21 22:33:55 526] TCPPacketReceiver::processPendingEvents(192.168.31.56:10047): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2018-06-21 22:33:55,528] - ==> [2018-06-21 22:33:55 526] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [2018-06-21 22:34:10,649] - 
  INFO root [2018-06-21 22:34:10,650] - -----------------------------------------------------------------------------------------


  INFO root [2018-06-21 22:34:10,651] - Resmgr::initialize: KBE_ROOT=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/
  INFO root [2018-06-21 22:34:10,651] - Resmgr::initialize: KBE_RES_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/res/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/res/
  INFO root [2018-06-21 22:34:10,651] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/bin/server/
  INFO root [2018-06-21 22:34:10,661] - NetworkInterface::recreateListeningSocket(EXTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:20013 )
  INFO root [2018-06-21 22:34:10,661] - NetworkInterface::recreateListeningSocket(EXTERNAL): address 192.168.31.56:20013, SOMAXCONN=128.
  INFO root [2018-06-21 22:34:10,664] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:10065 )
  INFO root [2018-06-21 22:34:10,664] - NetworkInterface::recreateListeningSocket(INTERNAL): address 192.168.31.56:10065, SOMAXCONN=128.
  INFO root [2018-06-21 22:34:10,664] - server-configs:
  INFO root [2018-06-21 22:34:10,665] - 	internalAddr : 192.168.31.56:10065
  INFO root [2018-06-21 22:34:10,665] - 	externalAddr : 192.168.31.56:20013
  INFO root [2018-06-21 22:34:10,666] - 	componentID : 9000
  INFO root [2018-06-21 22:34:12,707] - ---- loginapp Version: 1.0.0. ScriptVersion: 0.1.0. Protocol: A16A7477E55D6AEDCD48345D2B50A665. Config: Debug X86. Built: 23:20:06 Jun 15 2018. AppID: 9000. UID: 12668. PID: 13020 ----
  INFO root [2018-06-21 22:34:12,708] - Server :        Intel(R) Core(TM) i5-2415M CPU @ 2.30GHz with 4194303kB RAM
  INFO root [2018-06-21 22:34:12,709] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2018-06-21 22:34:12,710] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [2018-06-21 22:34:13,057] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [2018-06-21 22:34:13,059] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [2018-06-21 22:34:13,060] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [2018-06-21 22:34:13,067] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [2018-06-21 22:34:13,069] - Script::install_py_dlls(): PyInit_select
 DEBUG root [2018-06-21 22:34:13,069] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [2018-06-21 22:34:13,071] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [2018-06-21 22:34:13,510] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [2018-06-21 22:34:13,618] - Script::install(): is successfully, Python=(3.4.2 (default, Jun 15 2018, 23:20:02) [MSC v.1913 32 bit (Intel)])!
  INFO root [2018-06-21 22:34:13,619] - Script::install(): KBExtra is successfully!
S_INFO root [2018-06-21 22:34:13,624] - onLoginAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2018-06-21 22:34:13,626] - TelnetServer server is running on port 31000
  INFO root [2018-06-21 22:34:14,685] - ---- loginapp is running ----
 DEBUG root [2018-06-21 22:34:14,686] - Components::process(): Request for the process of identity...
  INFO root [2018-06-21 22:34:21,844] - Components::findComponents: find logger(1)...
  INFO root [2018-06-21 22:34:21,926] - Components::findComponents: found logger, addr:192.168.31.56:10070
  INFO root [2018-06-21 22:34:21,926] - Components::addComponent: reset orderLog, uid=12668!
  INFO root [2018-06-21 22:34:21,927] - Components::addComponent[logger], uid=12668, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2018-06-21 22:34:21,930] - Components::findComponents: find dbmgr(2)...
  INFO root [2018-06-21 22:34:22,032] - Components::findComponents: found dbmgr, addr:192.168.31.56:10071
  INFO root [2018-06-21 22:34:22,033] - Components::addComponent: reset orderLog, uid=12668!
  INFO root [2018-06-21 22:34:22,033] - Components::addComponent[dbmgr], uid=12668, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2018-06-21 22:34:22,033] - Components::findComponents: found dbmgr, addr:192.168.31.56:10071
  WARN root [2018-06-21 22:34:23,033] - Forwarding logs to logger[192.168.31.56:10070/0/0/0]...
 ERROR root [2018-06-21 22:34:54,229] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [2018-06-21 22:34:54,245] - The following logs sent to logger failed:
  WARN root [2018-06-21 22:34:54,246] - ==> [2018-06-21 22:34:54 226] TCPPacketReceiver::processPendingEvents(192.168.31.56:10070): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2018-06-21 22:34:54,246] - ==> [2018-06-21 22:34:54 230] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [2018-06-21 22:34:54,342] - TCPPacketReceiver::processPendingEvents(192.168.31.56:10071): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2018-06-21 22:34:54,346] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [2018-06-21 22:34:54,363] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [2018-06-21 22:34:54,368] - onLoginAppShutDown()
  INFO root [2018-06-21 22:34:54,368] - Shutdowner::onShutdown: shutting down
  INFO root [2018-06-21 22:34:54,369] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2018-06-21 22:34:54,369] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [2018-06-21 22:34:54,370] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [2018-06-21 22:34:54,370] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [2018-06-21 22:34:54,370] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [2018-06-21 22:34:54,371] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [2018-06-21 22:34:54,371] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [2018-06-21 22:34:54,371] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [2018-06-21 22:34:54,372] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [2018-06-21 22:34:54,392] - Script::uninstall(): is successfully!
 DEBUG root [2018-06-21 22:34:54,392] - ThreadPool::destroy(): starting size 2.
  WARN root [2018-06-21 22:34:54,692] - ThreadPool::destroy(): waiting for thread(2)[0x8d0198,0x8d0130,], try=1
  INFO root [2018-06-21 22:35:02,161] - 
  INFO root [2018-06-21 22:35:02,167] - -----------------------------------------------------------------------------------------


  INFO root [2018-06-21 22:35:02,169] - Resmgr::initialize: KBE_ROOT=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/
  INFO root [2018-06-21 22:35:02,169] - Resmgr::initialize: KBE_RES_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/res/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/res/
  INFO root [2018-06-21 22:35:02,170] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/bin/server/
  INFO root [2018-06-21 22:35:02,200] - NetworkInterface::recreateListeningSocket(EXTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:20013 )
  INFO root [2018-06-21 22:35:02,201] - NetworkInterface::recreateListeningSocket(EXTERNAL): address 192.168.31.56:20013, SOMAXCONN=128.
  INFO root [2018-06-21 22:35:02,208] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:10102 )
  INFO root [2018-06-21 22:35:02,208] - NetworkInterface::recreateListeningSocket(INTERNAL): address 192.168.31.56:10102, SOMAXCONN=128.
  INFO root [2018-06-21 22:35:02,209] - server-configs:
  INFO root [2018-06-21 22:35:02,209] - 	internalAddr : 192.168.31.56:10102
  INFO root [2018-06-21 22:35:02,209] - 	externalAddr : 192.168.31.56:20013
  INFO root [2018-06-21 22:35:02,209] - 	componentID : 9000
  INFO root [2018-06-21 22:35:04,316] - ---- loginapp Version: 1.0.0. ScriptVersion: 0.1.0. Protocol: A16A7477E55D6AEDCD48345D2B50A665. Config: Debug X86. Built: 23:20:06 Jun 15 2018. AppID: 9000. UID: 12834. PID: 13856 ----
  INFO root [2018-06-21 22:35:04,317] - Server :        Intel(R) Core(TM) i5-2415M CPU @ 2.30GHz with 4194303kB RAM
  INFO root [2018-06-21 22:35:04,317] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2018-06-21 22:35:04,318] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [2018-06-21 22:35:04,543] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [2018-06-21 22:35:04,545] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [2018-06-21 22:35:04,546] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [2018-06-21 22:35:04,553] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [2018-06-21 22:35:04,554] - Script::install_py_dlls(): PyInit_select
 DEBUG root [2018-06-21 22:35:04,554] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [2018-06-21 22:35:04,555] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [2018-06-21 22:35:04,701] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [2018-06-21 22:35:04,840] - Script::install(): is successfully, Python=(3.4.2 (default, Jun 15 2018, 23:20:02) [MSC v.1913 32 bit (Intel)])!
  INFO root [2018-06-21 22:35:04,840] - Script::install(): KBExtra is successfully!
S_INFO root [2018-06-21 22:35:04,845] - onLoginAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2018-06-21 22:35:04,846] - TelnetServer server is running on port 31000
  INFO root [2018-06-21 22:35:05,859] - ---- loginapp is running ----
 DEBUG root [2018-06-21 22:35:05,860] - Components::process(): Request for the process of identity...
  INFO root [2018-06-21 22:35:12,976] - Components::findComponents: find logger(1)...
  INFO root [2018-06-21 22:35:13,064] - Components::findComponents: found logger, addr:192.168.31.56:10100
  INFO root [2018-06-21 22:35:13,065] - Components::addComponent: reset orderLog, uid=12834!
  INFO root [2018-06-21 22:35:13,066] - Components::addComponent[logger], uid=12834, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2018-06-21 22:35:13,070] - Components::findComponents: find dbmgr(2)...
  INFO root [2018-06-21 22:35:13,170] - Components::findComponents: found dbmgr, addr:192.168.31.56:10103
  INFO root [2018-06-21 22:35:13,170] - Components::addComponent: reset orderLog, uid=12834!
  INFO root [2018-06-21 22:35:13,170] - Components::addComponent[dbmgr], uid=12834, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2018-06-21 22:35:13,171] - Components::findComponents: found dbmgr, addr:192.168.31.56:10103
  WARN root [2018-06-21 22:35:14,172] - Forwarding logs to logger[192.168.31.56:10100/0/0/0]...
